### PR TITLE
New release v0.0.6

### PR DESCRIPTION
**Pull request by holgerd77**
Ok, think we are finally ready for a revival release here 😄 , thanks @ryanio and @tomonari-t for all your work here!

@ryanio have added the note with the diagram update instructions to the README (we can separate/split this up in some future PR), think this will likely otherwise get lost. We should do on the devp2p library as well on next occasion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -305,6 +312,12 @@ also make use of subclasses of the ``Fetcher`` class that help fetch headers and
 - ``Node`` [**In Progress**] Represents the top-level ethereum node, and is responsible for managing the lifecycle of included services.
 - ``RPCManager`` [**In Progress**] Implements an embedded JSON-RPC server to handle incoming RPC requests.
 
+## Develper
```
Comment (ryanio) [inline README.md:315]:
```suggestion
## Developer
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,58 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+  PR [#126](https://github.com/ethereumjs/ethereumjs-client/pull/126)
+- Removed old fields `consensus` and `finality`, 
+  PR [#120](https://github.com/ethereumjs/ethereumjs-client/pull/120)
+
+**New Features**
+- Improved sync reliability,
+  PR [#133](https://github.com/ethereumjs/ethereumjs-client/pull/133)
+- Added `eth_getBlockTransactionCountByHash` RPC method,
+  PR [#125](https://github.com/ethereumjs/ethereumjs-client/pull/125)
+
```
Comment (tomonari-t) [inline CHANGELOG.md:58]: We also added  #131

Comment (holgerd77) [reply CHANGELOG.md:60]: Ah, sorry sorry, absolutely no idea how this could slip through 😕 , no bad intention, have updated!

Comment (tomonari-t) [reply CHANGELOG.md:60]: no problem haha ;) Thank you!

**Pull request merged**